### DISCUSSION TITLE
Release 4.0 Alpha

### DISCUSSION THREAD

**Discussion by ionosphere** [ekylibre/ekylibre#2584] [Announcements]
After a while, we are proud to annouce the 4.0 Alpha version with major update.

See https://demo.ekylibre.io for a quick review

It could be install from eky-core branch

**Comment by garthhumphreys** [comment]
Hi, is this available as a branch in the repo? I can't see it as a branch nor as a tag. Thanks

**Reply by ionosphere** [reply]
Hi,
It s the main branch. You also have a demo on https://demo-innovation.ekylibre.io
